package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "bot.js",
+  "dependencies": {
+    "mineflayer": "1.7.5",
+    "mineflayer-blockfinder": "0.0.5",
+    "mineflayer-navigate": "0.0.9"
+  },
+  "devDependencies": {
+    "flow-bin": "^0.33.0",
+    "flow-remove-types": "^1.0.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "flow; test $? -eq 0 -o $? -eq 2; flow-node"
```

Comment:
Oops, the `test` is redundant. The `exit 0` makes it so that if something fails, `npm` won't print its error messages.